tools/update-engine-size/src/main.rs

```diff
@@ -0,0 +1,74 @@
+use std::{
```

Comment:
Can be replaced with a simple shell script now, the initial idea was it was going to do more than this